migration/v0.17.x-v0.18.x/remove-velero-backupstoragelocation.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+[[ $(yq r "${CK8S_CONFIG_PATH}/sc-config.yaml" objectStorage.type) == "s3" ]] && STORAGE_TYPE_SC="aws" || STORAGE_TYPE_SC="gcs"
+[[ $(yq r "${CK8S_CONFIG_PATH}/wc-config.yaml" objectStorage.type) == "s3" ]] && STORAGE_TYPE_WC="aws" || STORAGE_TYPE_WC="gcs"
+
+"${here}/../../bin/ck8s" ops kubectl sc delete backupstoragelocation -n velero $STORAGE_TYPE_SC
+"${here}/../../bin/ck8s" ops kubectl wc delete backupstoragelocation -n velero $STORAGE_TYPE_WC
```

Comment:
If this is set to `none` we shouldn't set it to `gcs`. Then it should just exit the script. So I would rather have it more like this:

```suggestion
OBJECT_STORAGE_TYPE_SC=$(yq r "${CK8S_CONFIG_PATH}/sc-config.yaml" objectStorage.type)
if [[ ${OBJECT_STORAGE_TYPE_SC} == "s3" ]]; then
  STORAGE_TYPE_SC="aws"
elif [[ ${OBJECT_STORAGE_TYPE_SC} == "gcs" ]]; then
  STORAGE_TYPE_SC="gcs"
else
  exit 0
fi

"${here}/../../bin/ck8s" ops kubectl sc delete backupstoragelocation -n velero $STORAGE_TYPE_SC

OBJECT_STORAGE_TYPE_WC=$(yq r "${CK8S_CONFIG_PATH}/wc-config.yaml" objectStorage.type)
if [[ ${OBJECT_STORAGE_TYPE_WC} == "s3" ]]; then
  STORAGE_TYPE_WC="aws"
elif [[ ${OBJECT_STORAGE_TYPE_WC} == "gcs" ]]; then
  STORAGE_TYPE_WC="gcs"
else
  exit 0
fi

"${here}/../../bin/ck8s" ops kubectl wc delete backupstoragelocation -n velero $STORAGE_TYPE_WC
```

It will not be as neat as this script is, but I think that will better catch all possible combinations